state-chain/chains/src/btc.rs

```diff
@@ -173,29 +174,40 @@ impl Default for BitcoinFeeInfo {
 
 impl BitcoinFeeInfo {
 	pub fn new(sats_per_kilobyte: BtcAmount) -> Self {
-		Self { sats_per_kilobyte: max(sats_per_kilobyte, BYTES_PER_KILOBYTE) }
+		Self { sats_per_kilobyte: max(sats_per_kilobyte, BYTES_PER_BTC_KILOBYTE) }
 	}
 
 	pub fn sats_per_kilobyte(&self) -> BtcAmount {
 		self.sats_per_kilobyte
 	}
 
+	pub fn fee_per_utxo(&self, utxo: &Utxo) -> BtcAmount {
```

Comment:
nit: maybe we can rename this to ```fee_for_utxo``` since it takes in a utxo to calculate its fee

Reply:
good idea